fern-logger/src/config.rs

```diff
@@ -57,8 +57,20 @@ impl LoggerOutputConfigBuilder {
         self.target_filters = Some(
             target_filters
                 .iter()
-                .map(|f| f.to_string())
-                .collect::<Vec<String>>(),
+                .map(ToString::to_string)
+                .collect::<Vec<_>>(),
+        );
+        self
+    }
+
+    /// Sets a collection of exclusions of a logger output.
+    /// A message is logged only if one of the exclusions is *not* part of the log's metadata target.
+    pub fn target_exclusions(mut self, target_exclusions: &[&str]) -> Self {
```

Comment:
shouldn't this method be `#[must_use]` due to the newer clippy lints?

Reply:
It probably should...weird that clippy isn't failing.

Reply:
Ah, it's because `#[must_use]` is applied to the entire `LoggerOutputConfigBuilder` at line 22, so this is fine.